src/components/editor/sidebar.tsx

```diff
@@ -0,0 +1,56 @@
+import { Cog8ToothIcon } from "@heroicons/react/24/solid";
+import { EllipsisVerticalIcon } from "@heroicons/react/24/solid";
+
+function PageSelect({
+  page,
+  selectedPage,
+  setSelectedPage,
+}: {
+  page: any;
+  selectedPage: string;
+  setSelectedPage: any;
+}) {
+  const isSelected = selectedPage === page.id;
```

Comment:
Mmh instead of handling this like that I feel like we should simply pass a boolean called `isSelected` (this also mean removing the const under it). Rn we have to pass 2 pages to a tab for no reasons.